packages/ui/src/components/button.tsx

```diff
@@ -93,4 +93,4 @@ export const Button = withVariants<
         </TamaguiButton>
     )
   },
-)
+)
```

Comment:
There is a diff here but it shouldn't. I think it's that there is no blank line at the end of the file.